io.github.JaredTweed.SuperTuxAdvance.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "io.github.JaredTweed.SuperTuxAdvance",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "launch-sta",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--env=BRUX_AUDIO_BACKEND=pulseaudio",
+    "--env=SDL_AUDIODRIVER=pulseaudio",
+    "--device=all"
+  ],
+  "modules": [
+    {
+      "name": "lerc",
+      "buildsystem": "cmake",
+      "config-opts": [
+        "-DBUILD_SHARED_LIBS=ON"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/Esri/lerc/archive/refs/tags/v4.0.0.tar.gz",
+          "sha256": "91431c2b16d0e3de6cbaea188603359f87caed08259a645fd5a3805784ee30a0"
+        }
+      ]
+    },
+    {
+      "name": "supertux-advance",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/sta",
+        "cp -a sta-0.2.48/sta/* /app/sta/",
+        "install -Dm755 /app/sta/sta /app/bin/sta",
```

Comment:
so this is a binary? What happen on other architecture than the one it is built with?
Also why are you coying it?